src/bases/DataProviderBase.ts

```diff
@@ -55,6 +55,14 @@ abstract class DataProviderBase<T, O extends Options> {
 			if (this._data) {
 				observer.next(this._data);
 			}
+			return {
+				unsubscribe: () => {
```

Comment:
I had been getting notifications on closed objects. I'll look into this some more.